.github/workflows/build_and_test_debug_client.yml

```diff
@@ -216,7 +216,7 @@ jobs:
           go-version-file: '${{ github.workspace }}/go.mod'
 
       - name: Build Tun2Socks (required for Test OutlineAppleLib)
```

Comment:
*minor* Do you think we need to update the name from `Build Tun2Socks` to `Build Backend` in this file?